package.json

```diff
@@ -7,7 +7,7 @@
     "test": "mocha test/* --exit",
     "lint": "eslint *.js handlers/*.js test/**/*.js",
     "lint-fix": "eslint *.js handlers/*.js test/**/*.js --fix",
-    "start": "node caracal.js"
+    "start": "nodemon caracal.js"
```

Comment:
i think the start command should be via node and not nodemon.

Reply:
we could keep a separate command like `start:dev` or something similar to have hot reload

Reply:
Yes that's a good point